### ISSUE TITLE
Show only the Material Palette

### ISSUE THREAD

**Issue by chrisonline** [enricocid/ColorBox-library#2]
Is it possible to show only the material palette without the sliders view before?

**Comment by ramack** [comment]
Would also be interesting to have both, but the palette the view opening first. Or maybe even the 'last used' one...

**Comment by enricocid** [comment]
@chrisonline No, but I can add an option to let devs to choose which fragment to use
